go.mod

```diff
@@ -49,6 +49,7 @@ require (
 require (
 	github.com/ChainSafe/log15 v1.0.0 // indirect
 	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6 // indirect
+	github.com/aws/aws-sdk-go v1.42.20 // indirect
```

Comment:
Thinking about this, ideally it would be nice to have these toolings in other repositories, since it adds quite a few dependencies to Gossamer which actually doesn't need them. We can do this later before a v1 release I guess, since there are other tools as well.

Reply:
Yea I'm not sure about nested go.mod/go.sum files.

Reply:
I don't mind moving this stuff to another repo as a clean up step.  The actions will still live on this repo, but it will have pull access to the repo that contains all this code.